src/constants/transformation/transform-service.ts

```
@@ -220,6 +220,7 @@ export class TransformService {
             MemberStatus: cohort.cohortMemberStatus || 'active',
             AcademicYearID: cohort.academicYearId,
             CohortMemberID: cohort.cohortMemberId,
+            StatusReason:cohort.statusReason,
```

ESLint

[error] Insert `·`

(prettier/prettier)